app/src/main/java/org/p2p/wallet/moonpay/extensions/SellStubs.kt

```diff
@@ -38,4 +40,13 @@ object SellStubs {
         moonpayDepositWalletAddress = Constants.SOL_MINT.toBase58Instance(),
         updatedAt = emptyString()
     )
+
+    val TRANSACTION_DETAILS = SellTransactionViewDetails(
```

Comment:
Do we use it in code ?, If it was created for developing feature. I think we should remove this from code. Or use just in test package at all )

Reply:
yes, i use when i need to open some particular sell screens without any proxies like Charles